package.json

```diff
@@ -15,8 +15,7 @@
     "react-scripts": "2.1.2",
     "reactstrap": "^6.5.0",
     "redux": "^4.0.1",
-    "redux-form": "^8.1.0",
-    "redux-persist": "^5.10.0"
+    "redux-form": "^8.1.0"
```

Comment:
please leave `redux-persist` it is used

Reply:
I have removed it completely so evaluated people have less (new?) things to care about during assignment.

Also this requirements are moved to.
https://github.com/Selleo/washing-machine-lite